build.sbt

```diff
@@ -8,6 +8,7 @@ scalaVersion := "2.12.11"
 
 libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-sql" % "3.0.1",
+  "io.netty" % "netty-transport-native-epoll" % "4.1.46.Final",
```

Comment:
where is this required?

Reply:
I will remove this. 

For some reason, `pyspark` does not work with the `sbt publishLocal` package.  I will fix them later